src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -137,9 +137,11 @@
             }
             processConfiguration(configTopicsPojo);
         });
+        // Process initial configuration
+        processConfiguration(configTopics.toPOJO());
```

Comment:
calling `subscribe` already does this. Have a look at the implementation of subscribe, it calls the callback synchronously the first time.